diff --git a/ui/base/x/x11_display_util.cc b/ui/base/x/x11_display_util.cc
--- a/ui/base/x/x11_display_util.cc
+++ b/ui/base/x/x11_display_util.cc
@@ -343,7 +343,7 @@
 }
 
 base::TimeDelta GetPrimaryDisplayRefreshIntervalFromXrandr() {
-  constexpr base::TimeDelta kDefaultInterval = base::Seconds(1. / 60);
+  constexpr base::TimeDelta kDefaultInterval = base::Seconds(1. / 144);
   x11::RandR randr = x11::Connection::Get()->randr();
   auto root = ui::GetX11RootWindow();
   auto resources = randr.GetScreenResourcesCurrent({root}).Sync();
@@ -386,7 +386,7 @@
     if (refresh_rate == 0)
       continue;
 
-    return base::Seconds(1. / refresh_rate);
+    return base::Seconds(1. / 144);
   }
   return kDefaultInterval;
 }
